Source/Scene/WebMapServiceImageryProvider.js

```diff
@@ -174,6 +206,25 @@ define([
         });
     }
 
+    function requestImage(imageryProvider, col, row, level, request, interval) {
+        var dynamicIntervalData = defined(interval) ? interval.data : undefined;
+        var resource = imageryProvider._tileProvider._resource; // We actually want to set the time parameter within the tile provider.
+        var parameters = {};
+
+        if (defined(dynamicIntervalData)) {
+            if (!isNaN(dynamicIntervalData)){
+                parameters['time'] = interval.start.toString();
```

Comment:
Not exactly sure what is going on here? If `dynamicIntervalData` is a number, we just use the start time? Why is that?

Reply:
Ah, that may have been specific to my use case.  Let me look further.  Good catch, thanks.